.github/workflows/core_code_review.yaml

```diff
@@ -1,5 +1,12 @@
 name: PR - Core TF Validation
 
+# This pipeline start automatically when a PR is opened
```

Comment:
# This pipeline starts automatically when a PR is opened.

It is responsible for managing changes related solely to the NEW infrastructure. Therefore, it checks whether the changes have occurred only in the directories listed in "paths."

## NOTE: 'NEW infrastructure' refers to the new Terraform infrastructure located in the src/core/prod folder, which no longer requires the terraform.sh script to be applied.